openURL/src/comp.php

```diff
@@ -62,44 +60,63 @@
 		echo_message_to_screen(ERROR, "Could not open csv file - Process Stopped.");
 		exit;
     }
-	echo "Item ID" . "\t" . "Item Link" . "\t" . "Resource Link" . "\t" ."Item Title" . "\t" . "List Title" . "\t" . "Old Online Resource" . "\t" . "Old Online Link" . "\t" . "Outcome";
+	
+	// Write headersto audit file
+	$headers = [
+		"Item ID",
+		"Item Link", 
+		"Resource Link",
+		"Item Title",
+		"List Title",
+		"Old Online Resource",
+		"Old Online Link",
+		"Outcome"
+	];
+
+	echo "<table>";
+	writeToPage($headers);
+	writeToAuditFileOrExitOnFail($myfile, $headers);
```

Comment:
Writing the headers to both the page and the audit file.  Gathering them as an array so they are easy to read, and the `writeTo...` functions can decide on how to print them with separators/line endings.

Reply:
looks good :)